source/content/guides/autopilot/05-troubleshoot.md

```diff
@@ -109,10 +113,259 @@ Alternatively, you can add the plugin or theme to the **Excluded Updates** list
+
+</Accordion>
+
+## Merge to Dev
+
+<Accordion title="We could not merge the updates to the Dev environment due to an unexpected issue." id="merge-dev" icon="info-sign">
+
+### Issue
+
+Updates could not be merged to the Dev environment.
+
+### Diagnosis
+
+The most common reason for failure is due to a merge conflict. Changes were deployed in the Dev environment after Autopilot ran that resulted in a merge conflict. This error could also be because of a site-level PHP issue with `clear-cache` or `update.php` failing following a merge.
+
+### Solution
+
+If failure is because of a merge conflict, run Autopilot again with latest changes.
+
+If the issue is because of a PHP or CMS issue, resolve any issues causing `clear-cache` or `update.php` steps to fail, and manually merge again, or re-run Autopilot.
+
+</Accordion>
+
+## Cloned Content
+
+<Accordion title="We could not clone the environments due to an unexpected error." id="cloned-content-failed" icon="info-sign">
+
+### Diagnosis
+
+The most common reason for clone content to fail is that drush/wr-cli clear cache failed at the end of a workflow.
+
+### Solution
+
+Ensure the commands work. You will not be able to see if the workflow failed, but CSEs will be able to confirm if there was an issue with Drush or WP-CLI by replicating the step with Terminus.
+
+Use the following steps:
+
+* For WordPress:
+
+`wp cache flush`
+
+* Drupal:
+
+`drush cache-rebuild`
+
+</Accordion>
+
+## Uncommitted Changes
+
+<Accordion title="We could not run Autopilot because there is unsaved work in the development environment. Commit or discard the change, and retry Autopilot." id="changes-uncommitted" icon="info-sign">
+
+### Issue
+
+Unsaved work in th development environment has prevented Autopilot from running.
+
+### Solution
+
+Commit or discard the change, and run Autopilot again.
+
+</Accordion>
+
+## CMS Missing
+
+<Accordion title="We could not run Autopilot because a CMS was not detected. Install Drupal or WordPress on this site, and retry Autopilot." id="missing-cms" icon="info-sign">
+
+### Issue
+
+A CMS was not detected.
+
+### Solution
+
+Install Drupal or WordPress on this site, and run Autopilot again. For more information on creating a new Drupal or WordPress site on Pantheon, refer to the [documentation](https://pantheon.io/docs/create-sites).
+
+</Accordion>
+
+## Frozen Site
+
+<Accordion title="We could not run Autopilot because the site is frozen due to inactivity." id="site-frozen" icon="info-sign">
+
+### Issue
+
+A Sandbox site is frozen due to inactivity.
+
+### Solution
+
+Unfreeze the site in the old dashboard before re-queueing updates. Note, unfreezing may take some time.
+
+</Accordion>
+
+## Framework is Not Supported
+
+<Accordion title="We could not run Autopilot because the site is running an unsupported framework." id="framework-not-supported" icon="info-sign">
+
+### Diagnosis
+
+Site is running a framework that is not Drupal, Drupal 8, or Wordpress.
```

Comment:
```suggestion
Site is running a framework that is not Drupal, Drupal 8, or WordPress. WordPess Multisite is currently unsupported.
```

Maybe fix my words here, but good to clarify that WPMS (I thin framework is `wordpress_network`) is unsupported atm.